src/vs/editor/contrib/suggest/suggestWidget.ts

```diff
@@ -62,6 +62,10 @@ export const editorSuggestWidgetForeground = registerColor('editorSuggestWidget.
 export const editorSuggestWidgetSelectedBackground = registerColor('editorSuggestWidget.selectedBackground', { dark: listFocusBackground, light: listFocusBackground, hc: listFocusBackground }, nls.localize('editorSuggestWidgetSelectedBackground', 'Background color of the selected entry in the suggest widget.'));
 export const editorSuggestWidgetHighlightForeground = registerColor('editorSuggestWidget.highlightForeground', { dark: listHighlightForeground, light: listHighlightForeground, hc: listHighlightForeground }, nls.localize('editorSuggestWidgetHighlightForeground', 'Color of the match highlights in the suggest widget.'));
 
+/**
+ * Suggest widget styles
+ */
+const editorSuggestWidgetDeprecatedClassName = "suggest-widget-deprecated";
```

Comment:
I am getting a `Unexternalized string found: "suggest-widget-deprecated"` on this line from hygiene. Advice?

Reply:
We use double quotes for nls string (natural language strings, needs externalisation for translation). Use single quotes for 'other' string.